src/config/index.js

```diff
@@ -1,4 +1,6 @@
-export default {
+const index = {
   secret:
         process.env.NODE_ENV === 'production' ? process.env.SECRET : 'secret'
 };
+
+export default index;
```

Comment:
is there a need for changing to this syntax?

Reply:
Not really, but this is actually the original we had before Habeeb's modification and since I already wrote a middleware using the original I just wanted to revert.